### DISCUSSION TITLE
异域镇魂曲 “文本”有字体错位或字体溢出的情况

### DISCUSSION THREAD

**Discussion by owl342655596** [Gamuxorg/bbs#315] [游戏或网站运行反馈]
我玩过3DM汉化的那个Windows版本，它的汉化没有这种情况，只有对话选项会字体错位或溢出，站里的这个不仅对话选项有问题连文本也有问题。对话选项倒不是什么问题，主要是文本溢出没法看。

**Comment by owl342655596** [comment]
它那个好像是  哪个括号老爷汉化组汉化v1.19

**Comment by owl342655596** [comment]
站里面那个汉化有图片撕裂问题，3dm那个则没有。

**Comment by owl342655596** [comment]
3dm的我曾经流程的打通关过，站里面的这个没法玩。

**Comment by shouhuanxiaoji** [comment]
异域镇魂曲是官方的汉化，可能有些bug，我们看一下，方便截图报一下错吗

**Comment by owl342655596** [comment]
> 异域镇魂曲是官方的汉化，可能有些bug，我们看一下，方便截图报一下错吗

官方的没有发布汉化好吗，官方只汉化到了62.69%。站里面的那个就是民间汉化的，有图片撕裂（建立人物属性图就有这个问题），文本超出或被掩盖的问题。

**Comment by owl342655596** [comment]
下面这个是官方汉化的链接https://translate.baldursgate.com/projects/11

**Comment by owl342655596** [comment]
官方只有62.69%，没有发布汉化。
